examples/client/client.c

```diff
@@ -1099,455 +1099,674 @@ static int ClientRead(WOLFSSL* ssl, char* reply, int replyLen, int mustRead,
+            "-u          UDP DTLSを使用する。\n"
+            "           -v 2 を追加指定するとDTLSv1, "
+                    "-v 3 を追加指定すると DTLSv1.2 (既定値),\n"
+        "           -v 4 を追加指定すると DTLSv1.3\n", 
+
+    },
+#endif
+
 #ifdef WOLFSSL_SCTP
-        "-G          Use SCTP DTLS,"
-                " add -v 2 for DTLSv1, -v 3 for DTLSv1.2 (default)\n",  /* 22 */
+        {
+            OPTION_TYPE_SCTP_DTLS, 'G',
+            "-G          Use SCTP DTLS,"
+                " add -v 2 for DTLSv1, -v 3 for DTLSv1.2 (default)\n",
+            "-G          SCTP DTLSを使用する。-v 2 を追加指定すると"
+                " DTLSv1, -v 3 を追加指定すると DTLSv1.2 (既定値)\n",
+    },
 #endif
 #ifndef NO_CERTS
-        "-m          Match domain name in cert\n",                      /* 23 */
+        {
+            OPTION_TYPE_MATCH_NAME, 'm',
+            "-m          Match domain name in cert\n",
+            "-m          証明書内のドメイン名一致を確認する\n",
+    },
 #endif
-        "-N          Use Non-blocking sockets\n",                       /* 24 */
+        {
+            OPTION_TYPE_NON_BLOCKING, 'N',
+            "-N          Use Non-blocking sockets\n",
+            "-N          ノンブロッキング・ソケットを使用する\n"
+    },
 #ifndef NO_SESSION_CACHE
-        "-r          Resume session\n",                                 /* 25 */
+        {
+            OPTION_TYPE_RESUME_SESSION, 'r',
+            "-r          Resume session\n", 
+            "-r          セッションを継続する\n",
+    },
 #endif
-        "-w          Wait for bidirectional shutdown\n",                /* 26 */
-        "-M <prot>   Use STARTTLS, using <prot> protocol (smtp)\n",     /* 27 */
+        {
+            OPTION_TYPE_WAIT_BIDIRECT_SHUTDOWN, 'w',
+            "-w          Wait for bidirectional shutdown\n",
+            "-w          双方向シャットダウンを待つ\n", 
+    },
+        {
+            OPTION_TYPE_STARTTLS, 'M',
+            "-M <prot>   Use STARTTLS, using <prot> protocol (smtp)\n",
+            "-M <prot>   STARTTLSを使用する, <prot>プロトコル(smtp)を"
+                                              "使用する\n",
+
+    },
 #ifdef HAVE_SECURE_RENEGOTIATION
-        "-R          Allow Secure Renegotiation\n",                     /* 28 */
-        "-i <str>    Force client Initiated Secure Renegotiation. If the\n"
+        {
+            OPTION_TYPE_RENEGOTIATION, 'R',
+            "-R          Allow Secure Renegotiation\n",
+            "-R          セキュアな再ネゴシエーションを許可する\n", 
+
+    },
+        {
+            OPTION_TYPE_FORCE_RENEGOTIATION, 'i',
+            "-i <str>    Force client Initiated Secure Renegotiation. If the\n"
         "            string 'scr-app-data' is passed in as the value and\n"
         "            Non-blocking sockets are enabled ('-N') then wolfSSL\n"
         "            sends a test message during the secure renegotiation.\n"
-        "            The string parameter is optional.\n", /* 29 */
+        "            The string parameter is optional.\n", 
+        "-i <str>    クライアント主導のネゴシエーションを強制する\n",
```

Comment:
Please translate and add Japanese message line corresponding to English message.